src/main/frontend/handler/global_config.cljs

```diff
@@ -45,10 +45,12 @@
 (defn restore-global-config!
   "Sets global config state from config file"
   []
-  (let [config-dir (global-config-dir)
-        config-path (global-config-path)]
```

Comment:
The whole function is stuck here. So mobile screen shows a loading forever.

Reply:
Thanks for noticing this issue with switching graphs